src/core/use-toaster.ts

```diff
@@ -14,6 +14,10 @@ export const useToaster = (toastOptions?: DefaultToastOptions) => {
 
     const now = Date.now();
     const timeouts = toasts.map((t) => {
+      if (t.duration === -1) {
```

Comment:
Wouldn't `Infinity` make this more clear?

Reply:
Great suggestion, thanks.